go/epochtime/init.go

```diff
@@ -26,14 +25,14 @@ const (
 var Flags = flag.NewFlagSet("", flag.ContinueOnError)
 
 // New constructs a new Backend based on the configuration flags.
-func New(ctx context.Context, tmService service.TendermintService) (api.Backend, error) {
+func New(tmService service.TendermintService) (api.Backend, error) {
 	backend := viper.GetString(CfgBackend)
 	switch strings.ToLower(backend) {
 	case tendermint.BackendName:
 		interval := viper.GetInt64(CfgTendermintInterval)
-		return tendermint.New(ctx, tmService, interval)
+		return tendermint.New(tmService, interval)
```

Comment:
This seems awkward if you also provide a way for the consensus backend to initialize epochtime. Maybe move initialization of an epochtime backend to consensus in advance of #1879?